catalog/app/components/Footer/Footer.tsx

```diff
@@ -80,7 +87,7 @@ const useStyles = M.makeStyles((t) => ({
       '0px -12px 24px 0px rgba(25, 22, 59, 0.05)',
       '0px -16px 40px 0px rgba(25, 22, 59, 0.07)',
       '0px -24px 88px 0px rgba(25, 22, 59, 0.16)',
-    ],
+    ].join(' '),
```

Comment:
```suggestion
    ].join(', '),
```
shouldn't these be separated with commas?